pkg/cmd/admin/prune/images.go

```diff
@@ -27,9 +28,19 @@ import (
 )
 
 const (
-	imagesLongDesc = `%s %s - prunes images`
 	// PruneImagesRecommendedName is the recommended command name
 	PruneImagesRecommendedName = "images"
+
+	imagesLongDesc = `Prune images no longer needed by the system due to age and status.
+
+Only a user with a cluster role %s or higher who is logged-in will be able to actually delete
```

Comment:
Specify here that the command defaults to a dry run and will only prune if given `--confirm`